.ci/pytorch/test.sh

```diff
@@ -446,6 +446,17 @@ test_perf_for_dashboard() {
             "${target_flag[@]}" --"$mode" --"$dtype" --backend "$backend" "$@" \
             --output "$TEST_REPORTS_DIR/${backend}_max_autotune_${suite}_${dtype}_${mode}_cuda_${target}.csv"
       fi
+      if [[ "$DASHBOARD_TAG" == *cudagraphs_low_precision-true* ]] && [[ "$mode" == "inference" ]]; then
+        # TODO: This has a new dtype called quant and the benchmarks script needs to be updated to support this.
+        # The tentative command is as follows. It doesn't work now, but it's ok because we only need mock data
+        # to fill the dashboard.
+        python "benchmarks/dynamo/$suite.py" \
+          "${target_flag[@]}" --"$mode" --quant --backend "$backend" "$@" \
+          --output "$TEST_REPORTS_DIR/${backend}_cudagraphs_low_precision_${suite}_quant_${mode}_cuda_${target}.csv" || true
+        # Copy cudagraph results as mock data, easiest choice?
```

Comment:
Does this mock data get persisted anywhere?  I'm wondering if we need to plan to clean it up once we start using actual data, so that this doesn't corrupt the historical records

Reply:
The mock data will be persisted on Rockset under their own entries `cudagraphs_low_precision`.  I guess once the real data becomes available, I can clean them up.  Or we could just ignore them actually and they will go out of focus after a while.